src/incubator/Dialog/index.tsx

```diff
@@ -216,6 +217,10 @@ const Dialog = (props: DialogProps, ref: ForwardedRef<DialogImperativeMethods>)
     <View testID={`${testID}.overlayFadingBackground`} absF reanimated style={overlayStyle} pointerEvents="none"/>
   );
 
+  if (!modalVisibility) {
```

Comment:
This solves the tests issue? I don't understand? 

Will this means the layout calculation will happen every time the visibility changes?